server/routes/ticket.py

```
@@ -302,5 +300,333 @@ def post(self):
+                "closed": closed_count,
+            }
+
+            return restful_response(
+                status="success",
+                message="Ticket stats fetched successfully",
+                data=stats,
+                status_code=200,
+            )
+
+        except Exception as e:
+            current_app.logger.exception(f"Error fetching ticket stats ")
```

Flake8 (7.3.0)

[error] f-string is missing placeholders

(F541)